discovery_engine_core/tooling/src/newscatcher.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 Xayn AG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, version 3.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use anyhow::{Context, Result};
+use xayn_discovery_engine_providers::{Client, HeadlinesQuery, Market};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let url = "https://api-gw.xaynet.dev".to_string();
+    let token = std::env::var("GATEWAY_TOKEN").context(
+        "Please provide the GATEWAY_TOKEN environment variable for the dev environment. \
+                  The token can be found in 1Password",
+    )?;
```

Comment:
in 1password this is called bearer token, and there are different tokens, also without a prefix the env name is likely to have conflicts with other env. what about the following name instead?
```suggestion
    let token = std::env::var("NEWSCATCHER_DEV_BEARER_AUTH_TOKEN").context(
        "Please provide the NEWSCATCHER_DEV_BEARER_AUTH_TOKEN environment variable for the dev environment. \
                  The token can be found in 1Password",
    )?;
```